src/dwc_mapping.R

```diff
@@ -47,6 +48,16 @@ raw_data %<>%
   # Have sensible (lowercase) column names
   clean_names()
 
+#' We need to integrate the DwC term `taxonID` in each of the generated files (Taxon Core and Extensions).
+#' For this reason, it is easier to generate `taxonID` in the raw file. 
+#' First, we vectorize the digest function (The digest() function isn't vectorized. 
+#' So if you pass in a vector, you get one value for the whole vector rather than a digest for each element of the vector):
+vdigest <- Vectorize(digest)
+
+#' Generate `taxonID`:
+raw_data %<>% mutate(taxonID = paste("alien-plants-belgium:taxon:", vdigest (taxon, algo="md5"), sep=""))
```

Comment:
I would update this to:

    raw_data %<>% mutate(taxonID = paste("alien-plants-belgium", "taxon", vdigest (taxon, algo="md5"), sep=":"))

It's more logical to read.